soccer/gameplay/plays/offense/distraction1.py

```diff
@@ -0,0 +1,188 @@
+import robocup
+import standard_play
+import behavior
+import constants
+import main
+import skills.move
+import skills.capture
+import enum
+import evaluation
+import tactics.coordinated_pass
+import tactics.defense
```

Comment:
Defense isn't used so you can delete this. Defense is automatically added if you don't have a subbehavior with the name `defense`